### ISSUE TITLE
Typo Bug: SHFT_T is written instead of the proper SFT_T

### ISSUE THREAD

**Issue by squarology** [sboesebeck/ErgodoxLayoutGenerator#7]
Hey, Stephan, when I use shift for hold and send key when tapped (assign key ---> check shift ---> select key ---> check "if check modifier will be issued when held, key when tapped") that key is written like this SHFT_T(KC_BSPC) but there shouldn't be an H.

When I tried to compile with SHFT, I got an error until I checked the qmk readme and there was no H. It's towards the bottom of this section: https://github.com/jackhumbert/qmk_firmware#fun-with-modifier-keys.

**Comment by sboesebeck** [comment]
Oops, a typo. Fix that later today. Thanks for bringing that to me

/s

 From: squarology notifications@github.com To: sboesebeck/ErgodoxLayoutGenerator ErgodoxLayoutGenerator@noreply.github.com Subject: [sboesebeck/ErgodoxLayoutGenerator] Typo Bug: SHFT_T is written instead of the proper SFT_T (#7) Date: 30.05.16, 11:14

Hey, Stephan, when I use shift for hold and send key when tapped (assign key ---> check shift ---> select key ---> check "if check modifier will be issued when held, key when tapped") that key is written like this SHFT_T(KC_BSPC) but there shouldn't be an H.
When I tried to compile with SHFT, I got an error until I checked the qmk readme and there was no H. It's towards the bottom of this section: https://github.com/jackhumbert/qmk_firmware#fun-with-modifier-keys.—
